### PR TITLE
Rgbkb/keeb fixes

### PR DESCRIPTION
**Pull request by XScorpion2**
## Description

Small changes to update Mun & Sol 3 for latest QMK Master compatibility.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Fixed compile issue when CONSOLE_ENABLE = no 
* Set CONSOLE_ENABLE = no as default for improved BIOS compatibility
* Removed custom matrix for Mun as it's no longer compatible with QMK Master changes

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by zvecr — APPROVED** [review]

File: keyboards/rgbkb/common/common_oled.c [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #include "common_oled.h"
 #include "oled_driver.h"
 #include "rgb_matrix.h"
+#include "printf.h"
```
Comment (fauxpark) [inline keyboards/rgbkb/common/common_oled.c:13]:
```suggestion
#include "print.h"
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/rgbkb/common/common_oled.c (hunk 1 above)
Comment (XScorpion2) [reply keyboards/rgbkb/common/common_oled.c:13]: doesn't compile:
keyboards/rgbkb/common/common_oled.c: In function 'render_rgb_menu':
keyboards/rgbkb/common/common_oled.c:57:5: error: implicit declaration of function 'snprintf' [-Werror=implicit-function-declaration]

Comment (fauxpark) [reply keyboards/rgbkb/common/common_oled.c:13]: Should use `<stdio.h>` then.

Comment (XScorpion2) [reply keyboards/rgbkb/common/common_oled.c]: gotcha, saw the printf.h in the libs dir and thought that was the recommended approach over stdio.h. PR updated.

**Review by drashna — APPROVED** [review]

**Pull request merged**